core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -570,7 +564,7 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
       // Publish message on the request topic
       services.gpsDAO.publishMessages(
         importReadPubSubTopic,
-        List(MessageRequest(importId1.toString, testAttributes(importId1, legacyTestData.workspace.workspaceIdAsUUID)))
+        List(MessageRequest(importId1.toString, testAttributes(importId1)))
```

Comment:
I had [updated](https://github.com/broadinstitute/rawls/pull/3433/files#r2260503308) `testAttributes` to take in a workspaceId specifically for these legacy cases; previously it had always used `workspaceId` which is defined at the top as `testData.workspace.workspaceIdAsUUID`.  If there's no other legacy tests in this file you can revert it back to the way it was

Reply:
Fixed in [5858a5f](https://github.com/broadinstitute/rawls/pull/3441/commits/5858a5f9d285a2361a43c6411d21aa2443add599)